### PR TITLE
fix: latest changes

### PR DESCRIPTION
**Pull request by vsukhin**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by exu — APPROVED** [review]

File: internal/app/api/v1/tests.go [hunk 1]
```diff
@@ -436,7 +436,7 @@ func (s TestkubeAPI) UpdateTestHandler() fiber.Handler {
 		s.Log.Infow("updating test", "request", request)
 
 		var option *tests.Option
-		if request.Content != nil && (*request.Content) != nil && (*request.Content).Repository != nil && *(*request.Content).Repository != nil {
+		if request.Content != nil && (*request.Content) != nil && (*request.Content).Repository != nil {
```
Comment (vsukhin) [inline internal/app/api/v1/tests.go:439]: @nicufk why did you remove this check,  (*(*request.Content).Repository).Username can panic

Comment (nicufk) [inline internal/app/api/v1/tests.go:439]:
```suggestion
		if request.Content != nil && (*request.Content) != nil && (*request.Content).Repository != nil && *(*request.Content).Repository != nil {
```
